game-core/src/test/java/games/strategy/engine/chat/ChatPlayerPanelTest.java

```diff
@@ -0,0 +1,31 @@
+package games.strategy.engine.chat;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.awt.Dimension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ChatPlayerPanelTest {
+
+  @Mock private PlayerChatRenderer renderer;
+
+  @Test
+  public void setPlayerRendererWhenChatIsNull() {
+
+    ChatPlayerPanel chatPlayerPanel = new ChatPlayerPanel(null);
```

Comment:
Checkstyle fails with:
```
/triplea/game-core/src/test/java/games/strategy/engine/chat/ChatPlayerPanelTest.java:23:21: Variable 'chatPlayerPanel' should be declared final. [FinalLocalVariable]
```
```suggestion
    final ChatPlayerPanel chatPlayerPanel = new ChatPlayerPanel(null);
```